driver/generic/ili9xxx.py

```diff
@@ -0,0 +1,167 @@
+"""Generic ILI9xxx drivers.
+
+This code is licensed under MIT license.
+
+Adapted from:
+    https://github.com/rdagger/micropython-ili9341
+
+TODO:
+    * Usage information.
```

Comment:
Would you add some usage information? A few lines of example code to get the user started? (like [this](https://github.com/lvgl/lv_binding_micropython/pull/238#issuecomment-1280061740))

Reply:
added